awx/main/utils/common.py

```diff
@@ -161,6 +165,14 @@ def memoize_delete(function_name):
     return cache.delete(function_name)
 
 
+@memoize(ttl=3600 * 24)  # in practice, we only need this to load once at process startup time
+def get_event_partition_epoch():
+    from django.db.migrations.recorder import MigrationRecorder
+    return MigrationRecorder.Migration.objects.filter(
+        app='main', name='0124_event_partitions'
```

Comment:
@jladdjr if we end up reordering this migration (I'll bet exec envs beat us), we'll have to correct this label.